spring-cloud-dataflow-server-core/src/main/java/org/springframework/cloud/dataflow/server/config/DataFlowServerConfiguration.java

```diff
@@ -162,12 +164,28 @@ public Object postProcessAfterInitialization(Object bean, String s) throws Beans
 
 	@Bean
 	@ConditionalOnMissingBean
+	@ConditionalOnExpression("#{'${serverStore:redis}'.equals('db')}")
```

Comment:
should it be 'rdbms' instead of 'db'? To me 'db' could mean MongoDB, Cassandra or some other database. Also, shouldn't this be the default?